src/content/articles/en/how-to-slack.mdx

```diff
@@ -0,0 +1,187 @@
+
+At the same time, keep in mind that Slack is a place for short-lived communication. While the search functionality of Slack is pretty good and you can find things back really easily, you still need to know what you have to search for. You as the author might know but what about the others? Or even newbies?
+
+So make sure that you separate the information that has a long-term value from your day-to-day communication. Make sure that everything that you might need later is organized in a knowledge base. Tools like [Notion](https://www.notion.so/) or [Slite](https://slite.com/) might help you with that (or even [Confluence](https://www.atlassian.com/software/confluence) if you like to do it the painful way).
+
+Everything else can be communicated via Slack.
+
+## Your own Slack etiquette
+
+Slack is easy to use since it is just another messenger just like for example the Facebook Messenger. But can you therefore behave the same way? It is important that your work out a few rules and guidelines in your company. These guidelines will make sure that, while you have that team spirit and camaraderie, things do not get out of hands.
+
+<Note>
+  Make sure that you keep the guidelines in the knowledge base. This will be a
+  good place to start for newbies that enter your Slack. Even if they are only
+  contractors or single-channel guests.
+</Note>
+
+The Slack etiquette is something that looks different in every company here are a few guidelines that I really enjoyed so far.
+
+### Acknowledge messages
+
+If you receive a message from someone else with a task or request, make sure that you do not just ignore it. Even if you want to pick up the task only at a later time. Make sure that you acknowledge that you got the message and that you will pick it up soon.
+
+### Tell them if you leave
+
+If you are in a discussion but you are about to leave, make sure that you tell them. Nothing is worse than wondering why someone just stopped replying.
+
+### Do not spam your colleague
+
+If someone is not responding you immediately, it does not mean that they are ignoring you. They might be busy with another task, be in a meeting or just went to the kitchen to grab a coffee. So be patient and wait until they are back.
+
+### Don't @channel/@here just because you can
+
+It is tempting to push a notification to everybody just to make sure that everybody you want to reach are getting the notification. But most of the time you will send a notification more to people that are actually not interested.
+
+### Respect the channel's purpose
+
+If you have an exciting news to share, you might be triggered to share that in a channel that is actually dedicated for something else because you know that the people you want to reach are in there. This will to a long train of messages that do not belong in the channel and people actually working in there will have a hard time finding the things they need.
```

Comment:
```suggestion
If you have a piece of exciting news to share, you might be triggered to share that in a channel that is actually dedicated to something else because you know that the people you want to reach are in there. This will to a long train of messages that do not belong in the channel and people actually working there will have a hard time finding the things they need.
```